variants/backend-base/config/environments/development.rb

```diff
@@ -15,10 +15,6 @@
   RUBY
 end
 
-gsub_file "config/environments/development.rb",
-          "join('tmp', 'caching-dev.txt')",
-          'join("tmp/caching-dev.txt")'
```

Comment:
note: this is no longer matching because of the quotes, but Rubocop can automatically fix this anyway now